src/utils/dateHelpers.test.ts

```diff
@@ -137,23 +118,4 @@ describe('dateHelpers', () => {
       });
     });
   });
-
-  describe('formatISOString', () => {
```

Comment:
Det var visst to `formatISOString`-funksjoner: én i `formatDate.ts` og én `i dateHelpers.ts`. Den i `datehHelpers.ts` er den du tester her og den eksisterer fortsatt. Skal den det? 😄